releaser.yaml

```diff
@@ -0,0 +1,10 @@
+# Copyright 2024 Defense Unicorns
+# SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+
+flavors:
+  - name: upstream
+    version: 1.1.2
```

Comment:
Given us still using release-please we should add the comment annotations around all of these versions right?

Reply:
yes and edit the release please config to include this file

Reply:
Done!